Mage/Archivist/VaultValue.cs

```diff
@@ -32,7 +32,7 @@ public void SetData(string mediaType, JToken data)
 				MediaType = mediaType;
 
 				// Set data based on media type
-				Data = Tome.Conjure(JToken.Parse((string)data));
+				Data = data != null ? Tome.Conjure(JToken.Parse((string)data)) : null;
```

Comment:
`SetData` should never be called with null data, only with `JToken<null>`
If data is truly null, rather than `SetData` `Del` should be called